deploy/snapshot.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: snapshot.storage.k8s.io/v1alpha1
+kind: VolumeSnapshot
+metadata:
+  name: demo-snapshot
+spec:
+  snapshotClassName: csi-cstor
```

Comment:
Same as above name `csi-cstor-snapshotclass` just to differentiate with normal storageclass